randomUsernameAndPassword/assignment1/Q2.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+class Node:
+  def __init__(self, values):
+    assert(values)
+    head, tail = values[0], values[1:]
+    self.data = head
+    if tail:
+      self.next = Node(tail)
+    else:
+      self.next = None
+
+  def get_next(self):
```

Comment:
Please use direct access to attribute, this is common in Python
https://google.github.io/styleguide/pyguide.html?showone=Properties#Properties

What advantages do you see with the getters?

Reply:
Fixed